jumpscale/packages/vdc_dashboard/frontend/components/vdcSettings/VirtualMachine.vue

```diff
@@ -81,30 +81,22 @@
       title="Delete VM"
       :messages="deletionMessages"
       :wid="selectedvm"
-      @reload-vdcinfo="getVMSinfo"
+      @reload-vdcinfo="UpdateVms"
```

Comment:
I guess the real issue is that the implementation of  `UpdateVms` does not exist. so if added in the methods and bind the handler in the home call it will work fine.

```
UpdateVms() {
      this.$emit("reload-vdcinfo", {
        message: "VDC info has been reloaded successfully!",
      });
    },
```